bob-apps/Cargo.toml

```diff
@@ -51,7 +51,8 @@ default-features = false
 features = ["signal"]
 
 [dependencies.pearl]
-version = "0.14.0"
+# version = "0.14.0"
+git = "https://github.com/qoollo/pearl"
```

Comment:
New pearl is already available, so changes in this file should be reverted

Reply:
Done.